runtime/src/main/java/org/corfudb/runtime/collections/StreamSubscriptionMetrics.java

```diff
@@ -0,0 +1,35 @@
+package org.corfudb.runtime.collections;
+
+import io.micrometer.core.instrument.Timer;
+import org.corfudb.runtime.CorfuRuntime;
+
+import java.util.Optional;
+
+/**
+ * List of metrics captured for a stream listener.
+ * <p>
+ * Created by WenbinZhu on 11/23/20.
+ */
+public class StreamSubscriptionMetrics {
```

Comment:
Can you also add metrics on the consumer side. The kvstore already has issues where the consumer callbacks consume too much time, so I'm expecting we will need to track this as well.

Reply:
Yes we already have this metrics that records customer callback duration.